### PR TITLE
LPD-40345 testray.main.component.name should match with testray.team.{tema-name}.component.names in main test.properties file

### PR DESCRIPTION
**Pull request by ling-alan-huang**
https://liferay.atlassian.net/browse/LPD-40345

### CODE REVIEW

**Comment by ling-alan-huang** [comment]
ci:test:sf

**Comment by ling-alan-huang** [comment]
ci:test:relevant

**Comment by ling-alan-huang** [comment]
@ealonso Could you help to check my changes on `testray.main.component.name` in [test.proerpties](https://github.com/liferay-devtools/liferay-portal/pull/286/files#diff-015de4dffbfc96a304c1047bc564f2658552c3fed12f58ee446175d541ce8381R31)? Thanks.

File: modules/dxp/apps/enterprise-product-notification/test.properties [hunk 1]
```diff
@@ -28,4 +28,4 @@
 ## Testray
 ##
 
-    testray.main.component.name=Notification
\ No newline at end of file
+    testray.main.component.name=Notifications
```
Comment (ealonso) [inline modules/dxp/apps/enterprise-product-notification/test.properties:31]: @ling-alan-huang it should be User Notifications

**Comment by ling-alan-huang** [comment]
Thanks @ealonso I fixed.

**Comment by ling-alan-huang** [comment]
ci:forward

**Comment by ealonso** [comment]
@ling-alan-huang thanks you very much for working on it, it is going to help us a lot.

**Pull request closed without merging**